pkg/manager/client/discovery.go

```diff
@@ -78,6 +78,23 @@ func toObj(b []byte, groupVersion, kind string) (interface{}, error) {
 		}
 	}
 
+	if kind == "Secret" {
+		targets := []string{"token", "ca.crt"}
```

Comment:
Since those fields are implementation depends, so it's better if we just wipe out the whole `data` field.

Reply:
@yasker Updated. the whole data section is cleared.